src/features/todo-form/todo-form.tsx

```diff
@@ -0,0 +1,110 @@
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Todo, TodoSchema } from "../../shared/schemas/todo-schema";
+import { PRIORITIES } from "../../shared/schemas/priority-schema";
+import styles from "./todo-form.module.scss";
+import { FormField } from "./components/form-field/form-field";
+import { useTodoStore } from "../../shared/store/useTodoStore";
+import { capitalize } from "../../shared/util/capitalize";
+import { getTodayDate } from "../../shared/util/getTodayDate";
```

Comment:
Since the shared folder is a global folder, you can reduce the size of your imports by using absolute routes like:
`import { ... } from "@/shared/...."`